### PR TITLE
Added inf to be set for max range

### PR DESCRIPTION
**Pull request by dniewinski**

### CODE REVIEW

**Comment by tonybaltovski** [comment]
Other than style, LGTM.  Should this be enabled by default to comply with [REP117](http://www.ros.org/reps/rep-0117.html).  @mikepurvis any thoughts?

**Comment by mikepurvis** [comment]
For my information, why would you want this? All laser scanners have a maximum range— values in the scan falling beyond that point can't be trusted and should be excluded from any processing.

File: src/LMS1xx_node.cpp [hunk 1]
```diff
@@ -50,6 +51,7 @@ int main(int argc, char **argv)
 
   n.param<std::string>("host", host, "192.168.1.2");
   n.param<std::string>("frame_id", frame_id, "laser");
+  n.param<bool>("publish_max_range_as_inf", inf_range, false);
```
Comment (jasonimercer) [inline src/LMS1xx_node.cpp]: could this parameter name be changed to publish_max_range_as_inf ?

**Comment by servos** [comment]
@mikepurvis I'm not super sure why they decided to make max_range Inf but that is what is in REP117 so we have generally just gone along with it.

I suspect is mostly just for consistency when looking at the data. Ie if you are looking at the raw values you know, without comparing what the max_range is, that a measurement is invalid.

**Comment by mikepurvis** [comment]
I guess I'm fine with this, but if it's a short term hack to work around issues in older versions of autonomy, I wonder if it might make more sense to just have a separate external node that filters the scans?

**Comment by dniewinski** [comment]
Is this ok with everyone now?

**Comment by jasonimercer** [comment]
LGTM other than style. Missing a space after the if keyword and open braces should be on new lines

**Comment by tonybaltovski** [comment]
Same comments as Jason.  Maybe re-enable roslint tests.

**Pull request merged**